projects/igniteui-angular/src/lib/tabs/tabs.directive.ts

```diff
@@ -166,6 +172,26 @@ export abstract class IgxTabsDirective extends IgxCarouselComponentBase implemen
     protected scrollTabHeaderIntoView() {
     }
 
+    private setAttributes(igxName: string) {
+        this.items.forEach(item => {
+            this.setHeaderAttribute(item, 'aria-label', `${igxName}-tabitem-${NEXT_TAB_ID}`);
+            this.setHeaderAttribute(item, 'aria-controls', `${igxName}-tabpanel-${NEXT_TAB_ID}`);
+            this.setPanelAttribute(item, 'aria-labelledby', `${igxName}-tabitem-${NEXT_TAB_ID++}`);

```

Comment:
We are incrementing the ID counter event if the generated id is not used.

Reply:
Done